credentials/tls/certprovider/pemfile/builder.go

```diff
@@ -39,6 +40,8 @@ func init() {
 
 type pluginBuilder struct{}
 
+// ParseConfig parses the configuration data from JSON and returns a
+// certprovider.BuildableConfig for the PEM file watcher.
```

Comment:
Is vet failing for these?

It shouldn't.

This is an unexported type, so the docstrings won't appear anywhere.

These don't seem to provide any real value, so I'd just as soon delete them.  And I really hope we can configure the vet check better to not complain (if it is).

Reply:
Removed docstring from all methods with unexported types